.circleci/config.yml

```diff
@@ -33,6 +33,7 @@ jobs:
             pip install cython
             pip install .
             pip install scipy
+            pip install pandas
```

Comment:
it seems line 58 `git submodule update --init` should go here

Reply:
check